### ISSUE TITLE
Usage with server-side rendering?

### ISSUE THREAD

**Issue by savovs** [vkbansal/react-contextmenu#56]
Hi guys, having an issue with server-side rendering where window isn't defined, when I import the package like this: `import { ContextMenu, MenuItem, ContextMenuTrigger } from 'react-contextmenu';`

I get this error: 
```
ReferenceError: window is not defined
    at new GlobalEventListener (C:\Projects\Web\yupty-front\node_modules\react-contextmenu\modules\globalEventListener.js:49:5)
    at Object.<anonymous> (C:\Projects\Web\yupty-front\node_modules\react-contextmenu\modules\globalEventListener.js:53:19)
    at Module._compile (module.js:556:32)
    at Module._extensions..js (module.js:565:10)
    at Object.require.extensions.(anonymous function) [as .js] (C:\Projects\Web\yupty-front\node_modules\babel-register\lib\node.js:152:7)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
error Command failed with exit code 1.
```

It makes perfect sense because there's nothing to attach the listener to, because window isn't defined on the server. Not sure how to fix this though.

**Comment by vkbansal** [comment]
Thanks for raising the issue. The source code needs to be fixed for this. I'll do it ASAP.

**Comment by savovs** [comment]
No worries, this is an awesome repo!

**Comment by vkbansal** [comment]
published `v2.0.0-beta.2` with the fix.
